geobuf/encode.py

```diff
@@ -42,7 +42,10 @@ def encode(self, data_json, precision=6, dim=2):
 
         # print tf.MessageToString(data)
 
-        return data.SerializeToString()
+        if as_string:
+            return data.SerializeToString()
+        else:
+            return data
```

Comment:
@kputland instead of adding a flag to change the semantics of the method, I'd like to see the current method renamed from `encode` to `data`, returning only `self.data`, and then see the addition of a new `encode()` method that wraps `data()`, something like

```python
def encode(self, data_json, precision=6, dims=2):
    return self.data(data_json, precision=precision, dims=dims).SerializeToString()
```